pkg/services/dashboards/service/dashboard_service_integration_test.go

```diff
@@ -829,7 +766,7 @@ func TestIntegrationIntegratedDashboardService(t *testing.T) {
 						}
 
 						err := callSaveWithError(t, cmd, sc.sqlStore)
-						assert.Equal(t, dashboards.ErrDashboardFolderWithSameNameAsDashboard, err)
+						assert.NoError(t, err)
```

Comment:
maybe we should keep the other tests, and replace the assert with `NoError(...)`